zmon-actuator/src/test/java/org/zalando/zmon/actuator/ZmonMetricsFilterTest.java

```diff
@@ -54,10 +48,10 @@ public void test() throws InterruptedException {
             TimeUnit.MILLISECONDS.sleep(random.nextInt(500));
         }
 
-        assertThat(metricRegistry.getTimers().get("zmon.response.200.GET.hello")).isNotNull();
-        assertThat(metricRegistry.getTimers().get("zmon.response.503.GET.hello")).isNotNull();
+        assertThat(meterRegistry.get("zmon.response.200.GET.hello").timers()).isNotNull();
+        assertThat(meterRegistry.get("zmon.response.503.GET.hello").timers()).isNotNull();
 
-        String metricsEndpointResponse = restTemplate.getForObject("http://localhost:" + port + "/metrics",
+        String metricsEndpointResponse = restTemplate.getForObject("http://localhost:" + port + "/actuator/metrics/zmon.response.200.GET.hello",
                 String.class);
 
         logger.info(metricsEndpointResponse);
```

Comment:
What is this log statement doing here?

Reply:
this already existed from before. But I agree that it should be removed. Will do.